src/main/java/net/masterthought/cucumber/Configuration.java

```diff
@@ -94,4 +97,19 @@ public File getEmbeddingDirectory() {
         return new File(getReportDirectory().getAbsolutePath(), Configuration.EMBEDDINGS_DIRECTORY);
 
     }
+
+    /**
+     * @return Patterns to be used to filter out tags in the 'Tags Overview' chart. Returns an empty list by default.
+     */
+	public Pattern[] getTagsToExcludeFromChart() {
+		return tagsToExcludeFromChart;
+	}
+
+	/**
+	 * Stores the regex patterns to be used for filtering out tags from the 'Tags Overview' chart
+	 * @param patterns Regex patterns to match against tags
+	 */
+	public void setTagsToExcludeFromChart(Pattern... patterns) {
```

Comment:
so it means that at the same time you can see filtered tags xor all tags? so once you want to change this, report needs to be rebuilt?